native/mod_manager/mod_manager.c

```diff
@@ -1206,6 +1206,35 @@ static void mark_node_removed(nodeinfo_t *node)
         node->mess.num_remove_check = 0;
     }
 }
+static proxy_worker_shared *read_shared_by_node(request_rec *r, nodeinfo_t *node)
+{
+    void *sconf = r->server->module_config;
+    int i, port;
+    char *name = apr_pstrcat(r->pool, "balancer://", node->mess.balancer, NULL);
+    if (sscanf(node->mess.Port, "%u", &port)!=1)
+        return NULL; /* something is wrong */
+    proxy_server_conf *conf = (proxy_server_conf *)ap_get_module_config(sconf, &proxy_module);
+    proxy_balancer *balancer = (proxy_balancer *)conf->balancers->elts;
+    for (i = 0; i < conf->balancers->nelts; i++, balancer++) {
+        int j;
+        proxy_worker **workers;
+        if (strcmp(balancer->s->name, name))
+            continue;
+        workers = (proxy_worker **)balancer->workers->elts;
+        for (j = 0; j < balancer->workers->nelts; j++, workers++) {
+            proxy_worker *worker = *workers;
```

Comment:
I would consider writing it like this
```c
for (j = 0; j < balancer->workers->nelts; j++) {
    proxy_worker *worker = *(workers + j);
```

I believe it's cleaner.

Reply:
mod_balancer/proxy_util something cleaner...
+++
        proxy_worker **worker = (proxy_worker **)balancer->workers->elts;
        for (i = 0; i < balancer->workers->nelts; i++, worker++) {
+++

Reply:
My main "issue" was with the two expressions at the end, but leaving it as is works too, it's not a big deal.